tahrir_api/scripts/populatedb.py

```diff
@@ -0,0 +1,217 @@
+            'tester-04.png',
+            'tester-05.png',
+            'tester-06.png',
+            'tester-07.png',
+            'tester-08.png',
+            'tester-09.png',
+            'tester-10.png',
+        ]
+
+        USERS = ['sayanchowdhury', 'ralph', 'maxamillion', 'duffy']
+
+        SERIES = {
+            'copr-series': COPR_SERIES,
+            'fas-account-series': FAS_ACCOUNT_SERIES,
+            'badge-artist-series': BADGE_ARTIST_SERIES,
+            'tester-series': TESTER_SERIES,
+        }
+
+        TEAMS_SERIES_MAP = {
+            'infrastructure': ['copr-series', 'fas-account-series'],
+            'qa': ['tester-series'],
+            'design': ['badge-artist-series'],
+        }
+
+        SERIES_TAG_MAP = {
+            'copr-series': 'development, copr',
+            'fas-account-series': 'lifecycle, community',
+            'badge-artist-series': 'content, design, badges',
+            'tester-series': 'development, qa',
+        }
+
+        for name, series in TEAMS_SERIES_MAP.iteritems():
+            team = Team(
+                id=convert_name_to_id(name),
+                name=name,
+            )
+            DBSession.add(team)
+
+        for team_id, series in TEAMS_SERIES_MAP.iteritems():
+            for elem in series:
+                series_name = desc = ' '.join(elem.split('-')).title()
+                series = Series(
+                    name=series_name,
+                    description=desc,
+                    team_id=team_id,
+                    tags=SERIES_TAG_MAP.get(elem, '')
+                )
+                DBSession.add(series)
+
+        badge_series = {}
+        for slug, serie in SERIES.iteritems():
+            badge_series[slug] = []
+            for position, elem in enumerate(serie, 1):
+                name = ' '.join(elem.split('.png')[0].split('-')).title()
+                image = BADGES_PNGS_URL.format(badge_png=elem)
+                criteria = FEDORAHOSTED_URL.format(random.randint(1, 999))
+
+                badge = Badge(
+                    id=convert_name_to_id(name),
+                    name=name,
+                    image=image,
+                    description=name,
+                    criteria=criteria,  # TODO -- how should this work?
+                    issuer=issuer,
+                    tags=SERIES_TAG_MAP.get(slug, '')
+                )
+                DBSession.add(badge)
+                badge_series[slug].append(badge)
+
+                milestone = Milestone(
+                    position=position,
+                    badge_id=badge.id,
+                    series_id=slug
+                )
+                DBSession.add(milestone)
+
+        persons = []
+        for user in USERS:
+            person = Person(
+                nickname=user,
+                email='{username}@fedoraproject.org'.format(username=user),
+            )
+            DBSession.add(person)
+            persons.append(person)
+
+        transaction.commit()
```

Comment:
Why commit in the middle here? Not that it harms anything, but it doesn't seem to serve a purpose that is obvious to me. If there's a reason, I recommend adding a comment here.